bundle/src/test/groovy/be/orbinson/aem/groovy/console/extension/impl/DefaultExtensionServiceTest.groovy

```diff
@@ -153,7 +151,7 @@ class DefaultExtensionServiceTest {
     }
 
     private void runScriptWithExtensionService(ExtensionService extensionService) {
-        def binding = new Binding(out: new PrintStream(ByteStreams.nullOutputStream()))
+        def binding = new Binding(out: new PrintStream(OutputStream.nullOutputStream()))
```

Comment:
I think this requires Java 11+(https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/io/OutputStream.html#nullOutputStream()). Probably no issue to raise requirements, but this prevents running/building with Java 8